src/App.js

```diff
@@ -14,11 +14,87 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [...goodsFromServer],
+    currentTarget: [],
+  }
+
+toggleHandler = (good) => {
+  const copy = [...this.state.currentTarget];
+
+  if (!copy.includes(good)) {
+    this.setState({ currentTarget: [...copy, good] });
+  }
```

Comment:
You can add items directly to your 'currentTarget'. No need to create a copy of it